daemon/logdrivers_windows.go

```diff
@@ -6,6 +6,7 @@ import (
 	_ "github.com/docker/docker/daemon/logger/awslogs"
 	_ "github.com/docker/docker/daemon/logger/etwlogs"
 	_ "github.com/docker/docker/daemon/logger/fluentd"
+    _ "github.com/docker/docker/daemon/logger/gelf"
```

Comment:
Looks like you used spaces here for indentation; CI will probably fail because of that (Go / gofmt is very strict in formatting); you can manually fix it, or try running `gofmt -s -w daemon/logdrivers_windows.go`, which automatically reformats the file

Reply:
Sigh, sorry.  :(  I have defaults set for our standards and we don't use Go so likely it's whatever I have set to the default.  I'll make the changes.  I should have been more careful in my reading of the requirements.

Reply:
The tabs vs. spaces debate is probably the most lengthy one in history :joy: Go standardises the format, which may not be to everyone's liking, but at least saves a lot of time discussing which one to use ☺️